jupyter-notebooks/Supplemental_PAO1_FastICA_example.ipynb

```diff
@@ -0,0 +1,1467 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x1d340f487f0>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "sns.set(color_codes=True)\n",
+    "def plot_feature_weight_distributions(end_feature, dataframe):\n",
+    "    fig, ax = plt.subplots()\n",
+    "    for i in range(0, end_feature):\n",
+    "        sns.distplot(weight_matrix[:,i])\n",
+    "    ax.set_xlabel(\"Weight\")\n",
+    "    ax.set_ylabel(\"Number of genes at weight\")\n",
+    "    plt.show()\n",
+    "plot_feature_weight_distributions(50, weight_matrix)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note the similarity between this function and the ones we define in [`constants/gene_signature_definitions.py`](https://github.com/kathyxchen/PathCORE-analysis/blob/master/constants/gene_signature_definitions.py):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "def define_FastICA_gene_signature(std=2.5):\n",
```

Comment:
I am a bit confused as to why a couple of these functions are here and not imported with `pathcore` - they don't exist in pathcore? Maybe state that they are helper functions